pkg/volume/csi/csi_util.go

```diff
@@ -140,3 +137,34 @@ func hasReadWriteOnce(modes []api.PersistentVolumeAccessMode) bool {
 	}
 	return false
 }
+
+// getSourceFromSpec returns either CSIVolumeSource or CSIPersistentVolumeSource, but not both
+func getSourceFromSpec(spec *volume.Spec) (*api.CSIVolumeSource, *api.CSIPersistentVolumeSource, error) {
+	if spec == nil {
+		return nil, nil, fmt.Errorf("volume.Spec nil")
```

Comment:
nit: not `volume.Spec` just `spec` here,  since the object has no parent `volume` object in this function. ditto elsewhere

Reply:
@davidz627 Are you referring to the error message ?

Reply:
yes